test/functional/cursor_tests.js

```diff
@@ -4382,4 +4382,27 @@ describe('Cursor', function() {
       }
     }
   );
+
+  it('should return the correct count when find method used with collation set', function(done) {
```

Comment:
@mbroadst Is this the right location for this test? Or should we move this test to `collation_tests.js`

Reply:
+1 for moving to `collation_tests.js`